packages/aws-cdk-lib/rosetta/aws_s3_deployment/default.ts-fixture

```diff
@@ -4,6 +4,8 @@ import { Construct } from 'constructs';
 import * as s3deploy from 'aws-cdk-lib/aws-s3-deployment';
 import * as s3 from 'aws-cdk-lib/aws-s3';
 import * as ec2 from'aws-cdk-lib/aws-ec2';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as kms from 'aws-cdk-lib/aws-kms';
```

Comment:
Thanks for reviewing! @GavinZZ 

I'm following the guidelines on writing docs in the [Rosetta](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#rosetta) section. Without adding these lines, the new code snippets I added won't compile (e.g. Line 98 of [packages/aws-cdk-lib/aws-s3-deployment/README.md](https://github.com/aws/aws-cdk/pull/29540/files#diff-15a309d25a720b5fa37e0acf9fa11e0638de4302d6b9cc578e0efcb54110aeafR98), which uses `iam.PolicyStatement`). The section also recommends "Utilize the default.ts-fixture that already exists rather than writing new .ts-fixture files", so I'm updating the default ts-fixture file.

It seems that the CI (when I was making the PR) doesn't check if Rosetta compiles, but since this PR is mainly doc changes, I want to make sure the updates can be reflected on the CDK doc site.

Reply:
@kxue-godaddy thanks for getting back to me so quickly. I agree that the import statement for `iam` is necessary since it's used in the README file, but I don't think I see any usage of `kms` (other than the IAM actions). Would you please try removing the `kms` import statement and re-ran the command?

Reply:
@GavinZZ 

I removed the `kms` line and reran the command. It shows the following error:

```
aws-cdk-lib.aws_s3_deployment.Source#bucket-example.ts:24:18 - error TS2304: Cannot find name 'kms'.

24   encryptionKey: kms.Key.fromKeyArn(
```

The `kms` import line is needed by [packages/aws-cdk-lib/aws-s3-deployment/lib/source.ts](https://github.com/aws/aws-cdk/pull/29540/files#diff-dc0c9fe2615ec0f6de32500a9fbe2d5e5ba20743dded75bfc835707fd180651cR85), where I added an example for `Source.bucket` via the `@example` tag.

Reply:
oh nice, thank you. Didn't notice that there's a `kms` usage here.